packages/core/src/render3/query.ts

```diff
@@ -58,7 +54,7 @@ export class QueryState_ implements QueryState {
 
   constructor(deep?: QueryPredicate<any>) { this.deep = deep == null ? null : deep; }
 
-  track<T>(queryList: IQueryList<T>, predicate: Type<T>|any[], descend?: boolean): void {
+  track<T>(queryList: IQueryList<T>, predicate: Type<T>|string[], descend?: boolean): void {
     // TODO(misko): This is not right. In case of inherited state, a calling track will incorrectly
```

Comment:
can you take care of this comment as well? Or is it separate PR?

Reply:
Will do in a separate PR